packages/react-native-web/src/modules/ImageLoader/index.js

```diff
@@ -161,7 +211,20 @@ const ImageLoader = {
       }
     });
     return Promise.resolve(result);
+  },
+  // Resolves thr local blob for URIs loaded with `fetch`, otherwise just returns the URI
```

Comment:
```suggestion
  // Resolves the local blob for URIs loaded with `fetch`, otherwise just returns the URI
```

Reply:
I also think this needs to be a block comment

Reply:
I don't think I'm quite understanding this function and its purpose. If I just look at the logic, I would explain it like:

This method goes through all requests to find the request matching the URI argument. If that request is found, then the src of the image is returned. If nothing is found, then the URI is returned.

That seems weird. Why is a function returning two possible different things? I think it should just always return `src` or `null`. Then whatever is calling it can handle the `null` case (by falling back to the URI). That might help this small method to make more sense.

Also, the name of the method is very ambiguous (like a lot of methods in this code, probably not your fault) and could be named better.